docs/wiki/Getting started - Setup environment.md

```diff
@@ -0,0 +1,354 @@
+The repository is built so that you can create your own private 1:1 clone and be able to reuse the same concepts and features in your own environment such as GitHub.
```

Comment:
I'd avoid mentioning "private" given our primary option is to fork the public repository.

Reply:
Is it? The `Setup environment` would be the setup anybody would use to set it up at their customer. It's not per se intended for e.g. contributions.

Reply:
Yes but I think forking is still our primary suggestion and in that case the forked repo will be public as the main one. Maybe we can mention that later in the fork section.

Reply:
This confuses me a bit. Let's have a chat about that